Dockerfile

```diff
@@ -66,6 +66,7 @@ RUN --mount=type=cache,target=/go/pkg go list -mod=readonly all >/dev/null
 # builds kube-service-exposer-linux-amd64
 FROM base AS kube-service-exposer-linux-amd64-build
 COPY --from=generate / /
+COPY --from=embed-generate / /
```

Comment:
this feels like something was not properly generated... probably some bug Noel might have introduced while refactoring Kres

Reply:
Agreed. Not sure why it's getting added to this repo and not others we've rekresifyed, but let me see if I can figure out where it's coming from in kres.

Reply:
Looks like this isn't from kres getting github action support, but another PR previously. Adding the path to the version package like we do in kres.yaml here seems to fix: https://github.com/siderolabs/kres/blob/main/.kres.yaml#L47